libs/blocks/preflight/panels/accessibility.js

```diff
@@ -46,14 +47,39 @@ async function checkAlt() {
   if (!images) return;
 
   images.forEach((img) => {
+    if (img.classList.contains('accessibility-control')) return;
```

Comment:
it would be a little more efficient to just not select these images in the first place above - 
`const images = document.querySelectorAll('header img:not(.accessibility-control), main img:not(.accessibility-control), footer img:not(.accessibility-control)');`

Reply:
This was an omission for the initial iteration of the logic, but I can patch it up, thanks. I'll go with a `':is(header, main, footer) img:not(.accessibility-control)'` to keep things succint.